app/lib/efile/md/md502_calculator.rb

```diff
@@ -379,6 +380,44 @@ def calculate_line_20
         end
       end
 
+      def calculate_line_21
+        # Maryland state income tax
+        taxable_net_income = line_or_zero(:MD502_LINE_20)
+
+        if deduction_method_is_standard? && taxable_net_income >= 0
+
+          ranges = if taxable_net_income < 3_000
+                     [
+                       [0..1_000, 0, 0.02],
```

Comment:
so here we don't have to use `-Float::INFINITY` because we are checking already that the `taxable_net_income >= 0`

Reply:
Ooo yes originally added `-Float::INFINITY` but later learned that taxable net income will never be negative so removed it